README.md

```diff
@@ -94,6 +94,41 @@ class Test2 {
 } 
 ```
 
+#### Writing custom inspections
+
+Scapegoat supports custom inspections, for example to enforce a
+project-specific rule.
+
+How to write and test the inspection class is out of scope of this Readme, but
+you can take a look at the source code for
+[the inspections which are included in the scalac-scapegoat-plugin project](https://github.com/sksamuel/scalac-scapegoat-plugin/tree/master/src/main/scala/com/sksamuel/scapegoat/inspections).
+
+The `scapegoatCustomInspections` SBT SettingKey needs to list all the custom inspections by their fully-qualified class name. For example, add something like the following to your `build.sbt`:
+
+    scapegoatCustomInspections := List(
+      "my.inspections.InspectionOne",
+      "my.inspections.InspectionOne")
+
+The `scapegoatCustomInspectionsClasspath` SBT SettingKey needs to list the classpath entries at which your inspection classes can be loaded. If your inspections have come from a compiled JAR, then you can list its file location here. (There is not currently automatic support for using an Ivy-provided JAR dependency here. You may be able to do this by accessing the "update" SBT TaskKey. There is some code along these lines in [sbt-scapegoat](https://github.com/sksamuel/sbt-scapegoat/blob/ae4231d1341eeece323e111c757d57d904e66f7b/src/main/scala/com/sksamuel/scapegoat/sbt/ScapegoatSbtPlugin.scala#L41) which you can use for inspiration.)
```

Comment:
(The suggested code might work if the inspections are in a single SBT/Ivy imported JAR, but it wouldn't cope if the inspections have any transitive dependencies. I think there's not much `scapegoat` can do about that without help from SBT though.)